frontend/src/modules/Dashboard/Components/GroupCard.tsx

```diff
@@ -36,20 +39,45 @@ export const GroupCard = ({
     }
     setOpen(false)
   }
-
+  // returns color of background, button, and if newly matchable
+  function getColor(students: number, groups: number) {
+    //all students are matched
+    if (students === 0 && groups > 0) {
+      return [colors.paleviolet, colors.darkpurple, 'no']
+    }
+    //students are ready to be matched
+    else if (newStudents > 0 && groupsFormed > 0) {
+      return [colors.lightgreen, colors.darkgreen, 'no']
+    }
+    //NEWLY MATCHABLE
+    else if (newStudents > 1 && groupsFormed === 0) {
+      return [colors.lightgreen, colors.darkgreen, 'yes']
+    }
+    //only 1 student & 0 groups formed
+    else return [colors.lightyellow, colors.darkyellow, 'no']
+  }
+  const styleArray = getColor(
+    { newStudents }.newStudents,
+    { groupsFormed }.groupsFormed
+  )
   return (
-    <StyledContainer key={key}>
+    <StyledContainer color={styleArray[0]} key={key}>
+      {styleArray[2] === 'yes' && <StyledNewlyMatchable />}
       <StyledName>{name}</StyledName>
       <StyledRows>
         <StyledRow>
-          <StyledText>{submitted} Forms Submitted</StyledText>
+          <StyledPlusIcon></StyledPlusIcon>
+          <StyledText>{newStudents} News Student</StyledText>
+        </StyledRow>
+        <StyledRow>
+          <StyledGroupsIcon></StyledGroupsIcon>
+          <StyledText>{groupsFormed} Groups Formed</StyledText>
         </StyledRow>
-        <StyledRow></StyledRow>
       </StyledRows>
       <StyledButtons>
         <Button
```

Comment:
Does this button do anything yet?

Reply:
Not yet, but it should later lead to something of the form `/edit/350025`